src/declarations.js

```diff
@@ -33,3 +33,6 @@ declare var __FUNDING_ELIGIBILITY__: FundingEligibilityType;
 
 // Experiment Variable
 declare var __DISABLE_SET_COOKIE__: boolean;
+declare var __EXPERIENCE__: string;
+declare var __TREATMENT__: string;
+declare var __EXPERIMENTATION__: Object;
```

Comment:
Can we just add a type called `Experimentation` that looks like this?
```
{
    __EXPERIENCE__: string;
    __TREATMENT__: string;
}

Reply:
Then we only need to have this in the declaration file:
```
declare var __EXPERIMENTATION__: Experimentation;
```

& we can delete lines 36 & 37.